recipe/meta.yaml

```diff
@@ -30,51 +31,49 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - patch   # [not win]
+    - m2-patch    # [win]
   host:
+    - pip
+    - python
```

Comment:
cython >=3.0? we have plenty of cython versions less than that, although this is a bit of a nit

Reply:
Sure! Should I also add `setuptools >=51.0` because of [this](https://github.com/biolab/orange3/blob/3.36.2/pyproject.toml#L6)?

Reply:
I think that's less important because nearly all of our setuptools versions since ages are >=51

Reply:
pinning cython probably won't make a difference, the only thing it could take into account is if one of the other host dependencies had an upper bound on cython for some reason

Reply:
sometimes there are grey areas, or judgement calls, or things where it's maybe slightly better to do one thing or another but it in practice doesn't make a difference. See also [bikeshedding](https://thedecisionlab.com/biases/bikeshedding)

Reply:
Thanks for sharing! That was an interesting and enlightening read. I can admit I have commited bikeshedding a few many times, especially with my personal tasks. Lol